src/SFA.DAS.EmployerFinance.Web/Views/Transfers/Index.cshtml

```diff
@@ -9,6 +10,13 @@
 
 <h1 class="heading-xlarge">Your Transfers</h1>
 
+@if (Model.Data.RenderCreateTransfersPledgeButton)
+{
+<div class="govuk-button-group">
+    <a class="govuk-button" data-module="govuk-button" href="@Url.LevyMatchingTransfersAction("pledges")">Create a transfers pledge</a>
```

Comment:
Can we add an id attribute to this button, mostly so that automated tests can find it easily.